src/lib/stitching/gravity/stitching.ts

```diff
@@ -176,6 +179,32 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        descriptionFormatted: {
+          fragment: gql`
+            ... on ArtistSeries {
+              description
+            }
+          `,
+          resolve: async ({ description }, { format }) => {
+            if (!isExisty(description) || typeof description !== "string")
+              return null
+
+            const { type: formatType } = Format
+            const desiredFormat = formatType
+              ?.getValues()
+              ?.find((e) => e.name === format)?.value
+
+            return formatMarkdownValue(description, desiredFormat)
+
+            const formats = {
+              HTML: "html",
+              PLAIN: "plain",
+              MARKDOWN: "markdown",
+            }
+
+            return formatMarkdownValue(description, formats[format])
```

Comment:
Should this lower section have been deleted?

Reply:
😐 yes! following up